austation/code/modules/power/fuel_rods.dm

```diff
@@ -1,146 +1,291 @@
 /obj/item/twohanded/required/fuel_rod
-	var/conversion = "plutonium"// what does this rod convert to when depleted?
 	var/rad_strength = 500
 	var/half_life = 2000 // how many depletion ticks are needed to half the fuel_power (1 tick = 1 second)
 	var/time_created = 0
 	var/og_fuel_power = 0.20 //the original fuel power value
-	var/process =  FALSE
+	var/process = FALSE
+	// The depletion where Final_Depletion() will be called (and does something)
+	var/depletion_threshold = 100
+	// How fast this rod will deplete
+	var/depletion_speed_modifier = 1
+	var/depleted_final = FALSE // Final_Depletion should run only once
```

Comment:
is this variable needed for any reason? it seems pretty redundant considering the rod is deleted when final depletion is called

Reply:
Some depletions happen in a way that prevents the same `final_depletion()` being called more than once. But some doesn't. This variable is there to streamline (previous telecrystal rod was doing this with `grown` and `expended`) the `final_depletion()` process to make sure that it'll be actually called only once, for posterity.